tests/lib/rules/indent-legacy.js

```diff
@@ -23,19 +23,21 @@ const fixedFixture = fs.readFileSync(path.join(__dirname, "../../fixtures/rules/
 
 /**
  * Create error message object for failure cases with a single 'found' indentation type
- * @param {string} indentType indent type of string or tab
- * @param {array} errors error info
+ * @param {string} providedIndentType indent type of string or tab
+ * @param {array} providedErrors error info
  * @returns {Object} returns the error messages collection
  * @private
  */
-function expectedErrors(indentType, errors) {
-    if (Array.isArray(indentType)) {
-        errors = indentType;
-        indentType = "space";
-    }
+function expectedErrors(providedIndentType, providedErrors) {
+    let indentType;
+    let errors;
 
-    if (!errors[0].length) {
-        errors = [errors];
+    if (Array.isArray(providedIndentType)) {
+        errors = Array.isArray(providedIndentType[0]) ? providedIndentType : [providedIndentType];
```

Comment:
`providedIndentType` can be `[[]]`?   it seems to be string in comment.

Reply:
The function allows omitting the `indentType`, in which case the first argument gets used used as the errors.

```js
expectedErrors("space", [[]]);

// is the same as

expectedErrors([[]]);
```

Reply:
I see. Thanks for detailed explanation! 👍